source/XSharp/XSharp/Gen1/XS.FPU.cs

```diff
@@ -44,6 +44,35 @@ public static void FloatStoreAndPop(XSRegisters.Register32 register, bool isIndi
                 };
             }
 
+            public static void FloatStoreAndPop(XSRegisters.RegisterFPU register, bool isIndirect = false, int? displacement = null, XSRegisters.RegisterSize? size = null)
```

Comment:
Is this needed?

Reply:
It was needed for testing but It's not needed anymore sorry about that, I forgot. I'll go remove it.